administration/src/application/components/primitive-inputs/FileInputForm.tsx

```diff
@@ -48,6 +50,75 @@ type State = {
 type ValidatedInput = AttachmentInput
 type Options = {}
 type AdditionalProps = {}
+
+const Component = <I,>({
+  state,
+  setState,
+  required,
+  validate,
+}: FormComponentProps<State, AdditionalProps, Options> & {
```

Comment:
This was just extracted from below with the additional props "required" and "validate".